### DISCUSSION TITLE
"Waiting for a runner to pick up this job" When three jobs start simultaneously. Larger runner group supports 5 jobs..

### DISCUSSION THREAD

**Discussion by bhjelstrom** [community/community#38221] [Actions]
2022-11-03T17:34:39.3658615Z Requested labels: 
2022-11-03T17:34:39.3658691Z Job defined at:
2022-11-03T17:34:39.3658715Z Waiting for a runner to pick up this job...
2022-11-03T17:37:02.2883733Z Job is about to start running on the runner: 

![image](https://user-images.githubusercontent.com/13728935/199844567-3c0cc409-b7cc-4652-a489-8dac51cbe15d.png)

I'm pointing three action scripts at a new "larger runner" group that supports 5 jobs, but they have recently been running in sequence rather than in parallel.

Related to https://github.com/community/community/discussions/35490 but not the same.

**Comment by ossanna16** [comment]
Hi there @bhjelstrom and welcome to our community! Thank you for reporting this 🙂

**Comment by Santas** [comment]
We're seeing the same issue. Is Github able to reproduce this or is there any info we can provide?
